protocol/avdl/keybase1/block.avdl

```diff
@@ -76,5 +76,38 @@ protocol block {
   bytes getUserQuotaInfo();
   bytes getTeamQuotaInfo(TeamID tid);
 
+  record UsageStatRecord {
+    int64 write;
```

Comment:
Do you think it's worthwhile to specify a manual msgpack keys for these structs? Would help with compression on disk, although I guess thats not applicable in this case since we convert to another type. This just stuck out since we often specify these for new fields.

Something like
```
@mpackkey("w") @jsonkey("w")
```

Reply:
I honestly don't know. If you recommend it, I'll do it, but yeah this struct will never hit the disk.

Reply:
Let's skip